notebooks/demo2/setup_experiments.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    if SAMPLE_SIZE == "small":
</pre><p>If using an enum as suggested above, these if-else statements could be replaced by:</p><pre class="ql-syntax" spellcheck="false">SOURCE_DATA = SOURCE_DATA_PREFIX + f"/samples_{sample_size.value}/pdfs"
DESTINATION_DATA = DESTINATION_DATA_PREFIX + f"/{sample_size.name}/pdfs"
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>